WinUIGallery/AnimatedVisuals/LottieLogo1.cs

```diff
@@ -82,17 +82,17 @@ sealed class AnimatedVisual : IAnimatedVisual
             // Rectangle Path 1
             CompositionColorBrush ColorBrush_AlmostDarkTurquoise_FF00D1C1()
             {
-                return _c.CreateColorBrush(Windows.UI.Color.FromArgb(0xFF, 0x00, 0xD1, 0xC1));
+                return _c.CreateColorBrush(Microsoft.UI.ColorHelper.FromArgb(0xFF, 0x00, 0xD1, 0xC1));
```

Comment:
While I was briefly looking at unpackaged deployment, I happened across this, which was causing Windows.UI.Xaml.dll to be loaded.  We should be using Microsoft.UI color functionality, other than the Windows.UI.Color struct that does not require loading that DLL to be used.